dist/stat_data.py

```diff
@@ -202,6 +202,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_bytes_max', 'maximum bytes configured', 'no_clear,no_scale,size'),
     CacheStat('cache_bytes_other', 'bytes not belonging to page images in the cache', 'no_clear,no_scale,size'),
     CacheStat('cache_bytes_updates', 'bytes allocated for updates', 'no_clear,no_scale,size'),
+    CacheStat('cache_update_obsolete_calls', 'number of update obsolete check calls'),
```

Comment:
Not sure we need the stat here. Other than debugging this exact issue, I doubt if it will be useful in the long run. Maintaining stats could get expensive.